src/init.c

```diff
@@ -1263,11 +1263,17 @@ void jl_get_builtin_hooks(void)
     jl_interrupt_exception = jl_new_struct_uninit((jl_datatype_t*)core("InterruptException"));
     jl_boundserror_type    = (jl_datatype_t*)core("BoundsError");
     jl_memory_exception    = jl_new_struct_uninit((jl_datatype_t*)core("MemoryError"));
+    #ifdef _OS_LINUX_
+    #ifdef _P32
+        jl_float64_type->alignment = 4;
```

Comment:
Thanks, just checking. I saw the win32 complex erf failure, scrolled through the diff, and noticed this before I got to the test files and saw those were being skipped before.

Thanks for working on the docs here, this PR is quite a bit better about documentation than a lot of other comparable changes to internals have been.